src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -9,6 +9,7 @@ Release History
 * webapp, appserviceplan: adding pre-validation to the ASP and webapp create
 * webapp, functionapp: traffic-routing set fixed to use the correct actionHostName
 * functionapp: add slot support to functionapps
+* functionapp: add default creation of application insights on `az functionapp create`
```

Comment:
NIT: give this shows in the release documentation can we modify this to say "'az functionapp create' enables application insights by default'?